hoomd/md/pytest/test_rigid.py

```diff
@@ -146,10 +159,23 @@ def test_create_bodies(
     sim = simulation_factory(initial_snapshot)
 
     charges = [1.0, 2.0, 3.0, 4.0]
```

Comment:
You can reduce duplication by using a dictionary of optional arguments:
```suggestion
    optional_kwargs = {}
    if include_charge:
        optional_kwargs["charges"] = {"A": [1.0, 2.0, 3.0, 4.0]}
```